pre_commit/resources/hook-tmpl

```diff
@@ -136,7 +137,7 @@ def _pre_push(stdin):
             # ancestors not found in remote
             ancestors = subprocess.check_output((
                 'git', 'rev-list', local_sha, '--topo-order', '--reverse',
-                '--not', '--remotes={}'.format(remote),
+                '--not', '--remotes={}'.format(remote_name),
```

Comment:
you'll want to base on the latest master -- this line has been changed recently (so you'll probably conflict)